src/components/TransactionFlow/ConfirmTransaction.tsx

```diff
@@ -272,79 +255,84 @@ export const ConfirmTransactionUI = ({
 
       <RowWrapper>
         <ColumnWrapper>
-          <img src={sendIcon} alt="Send" />
+          <SIcon type="tx-send" alt="Send" height="25px" />
           {txType === ITxType.DEFIZAP
             ? translate('ZAP_CONFIRM_TX_SENDING')
             : translate('CONFIRM_TX_SENDING')}
         </ColumnWrapper>
-        <AmountWrapper>
-          <AssetIcon uuid={asset.uuid} size={'25px'} />
+        <ColumnWrapper>
           <Amount
-            fiatColor={COLORS.BLUE_SKY}
-            assetValue={`${bigify(amount).toFixed(6)} ${asset.ticker}`}
+            asset={{
+              amount: bigify(amount).toFixed(6),
+              ticker: asset.ticker,
+              uuid: asset.uuid
+            }}
             fiat={{
               symbol: getFiat(settings).symbol,
               ticker: getFiat(settings).ticker,
               amount: convertToFiat(bigify(amount), assetRate).toFixed(2)
             }}
           />
-        </AmountWrapper>
+        </ColumnWrapper>
       </RowWrapper>
       <RowWrapper>
         <ColumnWrapper>
-          <img src={feeIcon} alt="Fee" /> {translate('CONFIRM_TX_FEE')}
+          <SIcon type="tx-fee" alt="Fee" height="25px" />
```

Comment:
I would argue that using `img` for SVGs that we don't need to modify might be less of a PITA than what we are currently doing. It shouldn't be this difficult to use an image

Reply:
No objection to exporting img tag from Icon in the future.